tools/memap.py

```diff
@@ -10,10 +10,11 @@
 import argparse
 from prettytable import PrettyTable
 
-from tools.utils import argparse_filestring_type, \
```

Comment:
If I didn't drop the tool from the front of the import I get the following error runing memap.py as standalone from the tools folder (python memap.py ...)

> Traceback (most recent call last):
  File "memap.py", line 13, in <module>
    from tools.utils import argparse_filestring_type, \
ImportError: No module named tools.utils